spec/amp-cors-requests.md

```diff
@@ -37,7 +37,9 @@ by default.
 
 ## CORS Security in AMP
 
-This security protocol consists of two components: CORS origin and source origin restrictions.
+This security protocol consists of three components: `AMP-Same-Origin`, CORS origin and source origin restrictions.
```

Comment:
/cc @andreban @sebastianbenz for changes proposed to AMP CORS, to support in sample publisher as well.